src/modules/SignUpConfirmation.js

```diff
@@ -0,0 +1,28 @@
+import FormButton from '../FormButton';
+import { useHistory } from "react-router-dom";
+
+function SignUpConfirmation() {
+    let history = useHistory();
+
+    function handleClick() {
+        history.push('/SignIn');
+    }
+
+    return (
+        <>
+            <div className='header'>
+                <h2 className='header__title'>Congratulations!</h2>
+            </div>
+            <div class='sign-up-confirmation'>
+                <span className='sign-up-confirmation--span'>Your account has been successfully created.<br></br> Please continue to Log In.</span>
```

Comment:
please, never use <br>
the only exception may be if you making email template

Reply:
Ok, I removed `br`